client/http_client.go

```diff
@@ -164,11 +166,25 @@ func (slc *HttpClient) makeHttpRequest(url string, requestType string, requestBo
 		fmt.Fprintf(os.Stderr, "\n---\n[softlayer-go] Request:\n%s\n", hideCredentials(string(bs)))
 	}
 
-	resp, err := slc.HTTPClient.Do(req)
+	var resp *http.Response
+	for i:=1; i<=5; i++ {
```

Comment:
I don't like this hard coded retry count. Same for the actual wait time. We need these to be set as variables that can be changed by passing environment variables: `SL_API_RETRY_COUNT` and `SL_API_WAIT_TIME` if these environment variables are not set then I would be OK to defaul with 5 retries and 5 seconds wait.